src/queue/extras.rs

```diff
@@ -9,31 +9,40 @@ use std::time::{Duration, Instant};
 /// The extras for the task cells pushed into a queue.
 #[derive(Debug, Clone)]
 pub struct Extras {
+    pub(crate) start_time: Instant,
     /// The instant when the task cell is pushed to the queue.
     pub(crate) schedule_time: Option<Instant>,
     /// The identifier of the task. Only used in the multilevel task queue.
     pub(crate) task_id: u64,
-    /// The time spent on handling this task. Only used in the multilevel task
-    /// queue.
+    // The total time spent on handling this task.
     pub(crate) running_time: Option<Arc<ElapsedTime>>,
```

Comment:
It seems to me that this won't be shared across threads... So, an `Arc<ElapsedTime>` seems unnecessary?

Reply:
This is necessary because the task is moved when execute the task in worker inner

Reply:
Yes, it will be moved across thread boundaries. But it isn't modified concurrently by different threads, is it?

Reply:
No, it won't be visited by different thread concurrency. Use arc just to make the compiler happy. 😅

Reply:
Ah, I see. And I find it actually possible to be modified concurrently. After the inner future runner finishes polling, the task may be picked by another thread. At this time, there can be more than one threads in the `handle` thread for the same task.